app.js

```diff
@@ -0,0 +1,353 @@
+/*global alert ,confirm ,window ,setInterval ,setTimeout ,document ,clearInterval*/
+    //  Get user info
+(function getUserInfo() {
+    'use strict';
+    var url = window.location.href,
+        userName,
+        userAge,
+        greetDate,
+        userNamePlace,
+        getTimeNow;
+
+    //  if game page has no data(string) in url, keep moving.
+    //  Otherwise, declare user's name and age variables.
+    if ("string" === typeof url.split("?")[1]) {
+
+        userName = url.split("?")[1].split("=")[1].split("&")[0];
+        userAge = url.split("?")[1].split("=")[2];
+
+        //  Greeting New Guest
+        greetDate = document.getElementById('greeting-date');
+        userNamePlace = document.getElementById('user-name');
+        getTimeNow = new Date().getHours();
```

Comment:
`getTimeNow` now equals `new ( Date().getHours() )`, ie: it is now the equivalent of
```js
var1 = Date().getHours()
getTimeNow = new var1
```

You should instead do the following:
```js
now = new Date();
hours = now.getHours();
```